docs/product-business-resource-planner-guide/src/main/asciidoc/AuthoringPlanningAssets/optimizer-score-constraints-guided-rules-proc.adoc

```diff
@@ -0,0 +1,37 @@
+[id='optimizer-score-constraints-guided-rules-proc']
+= Define score constraints using the guided rules designer
+:imagesdir: ../..
+
+
+To solve an optimization problem, you need to define score constraints that evaluate your solution.
+{PLANNER} integrates with the guided rules designer and provides score modifiers, which are used by the engine during the solving process.
+
+Score modifiers can be accessed in the action selector (+) of the *THEN* (right-hand side) section of a rule, provided the planning solution is defined within the project.
+
+Use the following procedure to create a new score constraint using the guided rules designer.
+
+
+.Prerequisite
+
+You have defined a https://docs.jboss.org/optaplanner/release/latest/optaplanner-docs/html_single/index.html#solutionClass[Planning Solution]. 
+// NOTE FOR SME REVIEWER: where/how is this defined in the Employee Rostering example? 
+
+.Procedure
+. Open the project *Assets* screen by clicking *Menu* -> *Projects*, and clicking on your project to open it. 
+. Click *Create New Asset* in the upper-right corner.
+. Select *Guided Rule* from the drop-down menu.
+. In the *Create new Guided Rule* window, enter a name for the rule and click *Ok*.
+. If you want to extend a score constraint you have already created, click the drop-down box next to *EXTEND* and select the rule from the list.
+. Click the plus icon (image:5686.png[]) on the right side of the `WHEN` section to add a condition.
+. Once the action is selected, Business Optimizer score input appears on the *THEN* (right-hand side) section of the rule.
++
+The following {PLANNER} actions are available in the guided rules designer:
+* *Modify a single score level* - use the action to modify only one score component (e.g. hard score)
+* *Modify multiple score levels* - use the action to modify multiple score components at the same time (e.g. hard and soft score)
```

Comment:
This nested list is not rendered correctly.

Reply:
fixed